internal/grpc/handler.go

```diff
@@ -94,10 +95,21 @@ func (h *handler) Invoke(fileDescriptorSets []*descriptor.FileDescriptorSet, add
 	return invocationEventHandler.Err()
 }
 
-func (h *handler) dial(address string) (*grpc.ClientConn, error) {
+func getNetworkAddress(address string) (string, string) {
+	networks := []string{"tcp", "unix"}
+	for _, network := range networks {
+		scheme := network + "://"
+		if strings.HasPrefix(address, scheme) {
+			return network, strings.TrimPrefix(address, scheme)
+		}
+	}
+	return "tcp", address
+}
+
+func (h *handler) dial(network, address string) (*grpc.ClientConn, error) {
```

Comment:
Can you revert this to the original function signature and instead have this call `getNetworkAddress` instead?